src/robot.py

```diff
@@ -62,11 +121,60 @@ def robotInit(self):
     # Add the deploy artifacts to the shuffleboard
     addDeployArtifacts()
 
+  def disabledPeriodic(self):
+    self.drivebase.stop()
+
```

Comment:
Maybe also stop the launcher when disabled.

Reply:
Not strictly necessary. The framework will stop motors while disabled. We call the stop here because he drivebase has some periodic handling it needs to do as well. Can be added to any other branch for consistency.